config/app.go

```diff
@@ -41,17 +44,27 @@ const (
 	// EnvCiBitbucketRepoName Bitbucket CI variable for repo name
 	EnvCiBitbucketRepoName = "BITBUCKET_REPO_SLUG"
 
+	// EnvCiGitlabMrId Name of environment variable for Gitlab merge request id
+	EnvCiGitlabMrId = "CI_MERGE_REQUEST_IID"
+	// EnvCiGitlabUrl Name of environment variable for Gitlab Base Url
+	EnvCiGitlabUrl = "GITLAB_BASE_URL"
+	// EnvCiGitlabPid Name of environment variable for Gitlab Project ID
+	EnvCiGitlabPid = "CI_MERGE_REQUEST_PROJECT_ID"
+
 	VcsGithub    = "github"
 	VcsBitbucket = "bitbucket"
+	VcsGitlab    = "gitlab"
 
 	CiCircleCi  = "circleci"
 	CiBitbucket = "bitbucket"
+	CiGitlab    = "gitlab"
 )
 
 // NotifierConfig holds configuration
 type NotifierConfig struct {
 	LogFile       string `mapstructure:"LOG_FILE"`
 	TagID         string `mapstructure:"TAG_ID"`
+	ProjectID     string `mapstructure:"PROJ_ID"`
```

Comment:
to reduce necessary cli args and fit better into structure, can you remove and load the `PROJ_ID` from API by using repo and owner? It would be also more natural for user the specify only owner and repo instead of the id
```
curl -XGET -H "Content-Type: application/json" --header "PRIVATE-TOKEN: $GITLAB_TOKEN" https://gitlab.com/api/v4/projects/pansen2%2Flearn-gitlab-ultimate-trial/
```

Reply:
the Gitlab CI can provide these information too 
```
CI_PROJECT_NAME=learn-gitlab-ultimate-trial
CI_PROJECT_NAMESPACE=pansen2
```

Reply:
@karlderkaefer see what you think of this please.

Reply:
very good